platform/features/listview/res/templates/listview.html

```diff
@@ -0,0 +1,68 @@
+<div ng-controller="ListViewController">
+  <table>
+    <!-- Table Header -->
+    <thead>
+        <tr>
+          <th>Icon</th>
```

Comment:
There's a mix of two-space and four-space indentation here (and late in the same file), should use four spaces for consistency with other HTML sources